### PR TITLE
BigFloat erfcx

### PR DESCRIPTION
**Pull request by simonbyrne**
For #75

### CODE REVIEW

**Comment by simonbyrne** [comment]
Ah, that will only work for 64-bit...

**Comment by simonbyrne** [comment]
Okay, that should work for both.

**Comment by ararslan** [comment]
The nightly macOS failures seem to be due to an issue with BinDeps, but 32-bit Windows nightly is failing a test.

**Comment by simonbyrne** [comment]
Ah, it was the windows `Clong` problem...

**Comment by ararslan** [comment]
Nice, looks like CI is happy now.

File: src/erf.jl [hunk 1]
```diff
@@ -281,3 +281,31 @@ function erfcinv(y::Float32)
 end
 
 erfcinv(x::Integer) = erfcinv(float(x))
+
+# MPFR has an open TODO item for this function
+# until then, we use [DLMF 7.12.1](https://dlmf.nist.gov/7.12.1) for the tail
+function erfcx(x::BigFloat)
```
Comment (stevengj) [inline src/erf.jl]: Shouldn't it work (with minor modifications) for `Complex{BigFloat}` too?   I guess we don't have a complex `erfc` function for bigfloat, though.

Comment (simonbyrne) [reply src/erf.jl]: Not in its current form, but it probably could be adapted.

**Comment by putianyi889** [comment]
Seems like the connection part is missed.
```julia
Plots.plot(20000:40000,erfcx.(BigFloat(20000):40000), yaxis=:log)
```
download

**Comment by stevengj** [comment]
Can you file a separate issue?

**Pull request merged**